bundle/tests/scorecard/kuttl/deployment-strategy/04-remove-deployment.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: app.stacks/v1beta1
+kind: RuntimeComponent
+metadata:
+  name: deployment-strategy-rc
+spec:
+  # Add fields here
+  replicas: 3
+  deployment:
```

Comment:
@halim-lee Tanks for updating the tests. The flow looks good. Wondering whether `deployment` field can be removed altogether here? or does kuttl require it to be set this way?

If it can be removed, please do the same for updateStrategy field. Thanks.

Reply:
kuttl test updates what is specified in the yaml file. If I remove deployment field, it wouldn't know deployment should be removed - so I made the deployment field empty.